neuralmonkey/decoders/transformer.py

```diff
@@ -59,6 +59,8 @@ def __init__(self,
             data_id=data_id,
             max_output_len=max_output_len,
             dropout_keep_prob=dropout_keep_prob,
+            # from the "dark slide"
```

Comment:
tohle vyhoď a dej ten label_smoothing do parametrů initu - chceme to štelovat z konfigu